cfme/tests/cloud/test_stack.py

```diff
@@ -16,41 +19,62 @@ def pytest_generate_tests(metafunc):
     testgen.parametrize(metafunc, argnames, argvalues, ids=idlist, scope='module')
 
 
-def set_grid_view(name):
-    bg = ButtonGroup(name)
-    sel.force_navigate("my_settings_default_views")
-    default_view = bg.active
-    if(default_view != 'Grid View'):
-        bg.choose('Grid View')
-        sel.click(form_buttons.save)
-
-
 @pytest.fixture(scope="function")
 def stack(setup_provider, provider, provisioning):
-    set_grid_view("Stacks")
-    stackname = provisioning['stack']
-    stack = Stack(stackname)
-    return stack
+    return Stack(provisioning['stack'])
 
 
 @pytest.mark.tier(3)
 def test_security_group_link(stack):
-    stack.nav_to_security_group_link()
+    navigate_to(stack, 'Details')
+    # expand accordion by text
+    accordion.click('Relationships')
+    # click anchor by title attribute, the anchor text itself could change
+    sel.click('//*[@id="stack_rel"]//a[@title="Show all Security Groups"]')
```

Comment:
So this is actually accordion navigation, and can definitely be improved.  The accordion field text changes (it puts the number of the item after, so Security Groups (#)).

Was not actually aware that you want to avoid sel actions within the tests, will re-evaluate this approach.